usr/share/regolith-look/blackhole/gnome-terminal

```diff
@@ -0,0 +1,21 @@
+#define XRES_GNOME_TERMINAL
+
+gnome.terminal.color0:    color_black
```

Comment:
`color_black` needs to be defined above in the `color` file.  The reason it's not working for me is that it's evaluating to `color_black` rather than an actual color value:

```
$ xrescat gnome.terminal.color0
color_black
```